tests/fileTrashTest.js

```diff
@@ -60,106 +143,228 @@ describe('oc.fileTrash', function () {
     oc = null
     done()
   })
-
-  it('should have the trashbin capability set', function (done) {
-    if (!trashEnabled) {
-      pending()
-    }
-    oc.getCapabilities().then(cap => {
-      expect(cap.capabilities.dav.trashbin).toEqual('1.0')
-      done()
-    }).catch(error => {
-      fail(error)
-      done()
+  fdescribe('when deleting files and folders', function () {
+    beforeAll(function (done) {
+      Promise.all(setGeneralInteractions(provider)).then(done, done.fail)
     })
-  })
-
-  describe('when empty', function () {
-    it('should list no items ', function (done) {
+    afterAll(function (done) {
+      provider.removeInteractions().then(done, done.fail)
+    })
+    fit('should have the trashbin capability set', function (done) {
       if (!trashEnabled) {
         pending()
       }
-      oc.fileTrash.list('/').then(trashItems => {
-        expect(trashItems.length).toEqual(1)
-        expect(trashItems[0].getName()).toEqual(userId)
+      oc.getCapabilities().then(cap => {
+        expect(cap.capabilities.dav.trashbin).toEqual('1.0')
+        done()
+      }).catch(error => {
+        fail(error)
         done()
       })
     })
   })
 
-  describe('when a folder is deleted', function () {
-    let testFolder, testFile
-
-    beforeEach(function (done) {
-      if (!trashEnabled) {
-        pending()
-      }
-
-      const suffix = Math.random().toString(36).substr(2, 9)
-      testFolder = 'testFolder' + suffix
-      testFile = testFolder + '/file.txt'
-      oc.files.createFolder(testFolder).then(() => {
-        oc.files.putFileContents(testFile, '*').then(() => {
-          oc.files.delete(testFolder).then(() => {
-            done()
-          })
-        })
-      })
+  fdescribe('and when empty', function () {
```

Comment:
You could make it more clear. something like `when the trashbin is empty`

Reply:
Yes, but originally it was `and when empty`. So I'm not sure if we should change it or not https://github.com/owncloud/owncloud-sdk/blob/26bc93b166d001835a4995249af0bb207729818c/tests/fileTrashTest.js#L77